integration-tests/src/test/java/uk/gov/di/accountmanagement/api/AuthenticateIntegrationTest.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.di.accountmanagement.api;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import jakarta.ws.rs.core.Response;
+import org.junit.jupiter.api.Test;
+import uk.gov.di.accountmanagement.entity.AuthenticateRequest;
+import uk.gov.di.authentication.api.IntegrationTestEndpoints;
+import uk.gov.di.authentication.frontendapi.entity.LoginRequest;
+import uk.gov.di.authentication.frontendapi.entity.LoginResponse;
+import uk.gov.di.authentication.helpers.DynamoHelper;
+import uk.gov.di.authentication.helpers.RequestHelper;
+
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static uk.gov.di.authentication.shared.entity.SessionState.LOGGED_IN;
+
+public class AuthenticateIntegrationTest extends IntegrationTestEndpoints {
+
+    private static final String LOGIN_ENDPOINT = "/authenticate";
+    private final ObjectMapper objectMapper = new ObjectMapper();
+
+    @Test
+    public void shouldCallLoginEndpointAndReturn200WhenLoginIsSuccessful() throws IOException {
+        String email = "joe.bloggs+3@digital.cabinet-office.gov.uk";
+        String password = "password-1";
+        String phoneNumber = "01234567890";
+        DynamoHelper.signUp(email, password);
```

Comment:
this lambda is to validate the user credentials is there a reason it is calls signup?

Reply:
that's a test helper that creates a user account, need an existing account for the test.